spirv_msl.cpp

```diff
@@ -2229,6 +2278,82 @@ string CompilerMSL::to_sampler_expression(uint32_t id)
 	return samp_id ? to_expression(samp_id) : to_expression(id) + sampler_name_suffix;
 }
 
+// Checks whether the ID is a row_major matrix that requires conversion before use
+bool CompilerMSL::is_non_native_row_major_matrix(uint32_t id)
+{
+	// Natively supported row-major matrices do not need to be converted.
+	// Legacy targets do not support row major.
+	if (backend.native_row_major_matrix && !is_legacy())
```

Comment:
is_legacy doesn't really make sense on MSL? it's more of a GLSL/HLSL thing.

Reply:
Copy and pasted code. Good catch. Removed now.